frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -162,6 +162,67 @@ func ListTargets(ctx context.Context, dt []byte) (*targets.List, error) {
 	return l, nil
 }
 
+func parseLintOptions(checkStr string) (*linter.Config, error) {
+	checkStr = strings.TrimSpace(checkStr)
+	if checkStr == "" {
+		return &linter.Config{}, nil
+	}
+
+	parts := strings.SplitN(checkStr, ";", 2)
+	var skipSet []string
+	var errorOnWarn, skipAll bool
+	for _, p := range parts {
+		k, v, ok := strings.Cut(p, "=")
+		if !ok {
+			errors.Errorf("invalid check option %q", p)
```

Comment:
Missing return. (Weird if golint does not catch this)

Reply:
Definitely weird none of the linters caught this.